src/Compilers/CSharp/Portable/FlowAnalysis/NullableWalker.cs

```diff
@@ -860,7 +859,7 @@ private void EnterParameter(ParameterSymbol parameter, TypeSymbolWithAnnotations
 
 #region Visitors
 
-        public override BoundNode VisitIsPatternExpression(BoundIsPatternExpression node)
+        public override BoundNode? VisitIsPatternExpression(BoundIsPatternExpression node)
```

Comment:
>BoundNode? [](start = 24, length = 10)

It looks like we always return parameter that isn't nullable. Why was this change necessary?